api/src/components/coauthor/controller.ts

```diff
@@ -253,13 +253,19 @@ export const updateConfirmation = async (
             });
         }
 
-        // Is the publication in draft?
+        // Is the publication in locked mode?
         if (publication.currentStatus === 'LIVE') {
             return response.json(403, {
                 message: 'This publication is LIVE and therefore cannot be edited.'
```

Comment:
This message needs to be changed because the confirmation cannot be processed while the currentStatus is LIVE or DRAFT